tools/container/runtime/runtime.go

```diff
@@ -89,6 +90,13 @@ func Flags(opts *Options) []cli.Flag {
 			EnvVars:     []string{"NVIDIA_RUNTIME_SET_AS_DEFAULT", "CONTAINERD_SET_AS_DEFAULT", "DOCKER_SET_AS_DEFAULT"},
 			Hidden:      true,
 		},
+		&cli.BoolFlag{
+			Name:        "runtime-enable-cdi",
```

Comment:
Let the flag be passed as `--enable-cdi`. The envar and the Option variable can have the `Runtime/RUNTIME` prefix. This is consistent with the other flags like `socket`, `restart-mode` etc
```suggestion
			Name:        "enable-cdi",
```

Reply:
Let's try to be consistent with the option for the `nvidia-ctk runtime configure` command.

Reply:
The problem is that we already have a `cdi-enabled` / `enable-cdi` flag defined in the toolkit options which triggers the generation of a CDI spec: https://github.com/NVIDIA/nvidia-container-toolkit/blob/8d869acce5643ed0de0a358f605bf14f8bc05fca/tools/container/toolkit/toolkit.go#L173-L179

Any ideas on what the name of the new flag should be?

Reply:
Thanks for the clarification.

One question I would have is whether we need a separate flag? Would it not make sense for the same flag / envvar (e.g. `CDI_ENABLED`) to be use to also trigger enabling CDI in the runtime? Do older versions of containerd complain about invalid config options?

If we feel we want finer control, then I'm ok to use `RUNTIME_ENABLE_CDI` or `ENABLE_CDI_IN_RUNTIME` to be used. (maybe the latter).